docs/user-documentation/moderne-platform/how-to-guides/writing-and-installing-recipes.md

```diff
@@ -0,0 +1,38 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Writing and Installing Recipes
+
+[OpenRewrite](https://docs.openrewrite.org/) is a framework for writing and running code transformations. A new code transformation, called [recipes](https://docs.openrewrite.org/concepts-and-explanations/recipes) can be written
+and uploaded to your instance of the Moderne Platform. This is done in 3 steps:
+
+1. Writing custom recipes
+2. Publish artifact containing the custom recipes
+3. Importing Recipes
+
+## Writing Recipes
+Before writing a new recipe, check the [recipe catalogue](https://docs.openrewrite.org/recipes) to see if an existing recipe is available. For transformations that aren't covered by the existing catalogue, new recipes can be written in [declarative YAML](https://docs.openrewrite.org/concepts-and-explanations/recipes#declarative-recipes),
```

Comment:
maybe use an unordered list to enumerate the 3 types of recipes?

Reply:
This look better, thanks for the suggestion